STP-3186 (1.0): correcting broken fenced code blocks (#1477)
Cray-HPE/docs-csm: operations/node_management/Adding_a_Liquid-cooled_blade_to_a_System.md

Use boot orchestration to power on and boot the nodes. Specify the appropriate BOS template for the node type.

16. Determine how the BOS Session template references compute hosts.
    

>     Typically, they are referenced by their "Compute" role. However, if they are referenced by xname, then these new nodes should added to the BOS Session template.

Reviewer: ```suggestion
    Typically, they are referenced by their `Compute` role. However, if they are referenced by component name (xname), then these new nodes should added to the BOS Session template.
```

    
    ```bash
    ncn-m001# BOS_TEMPLATE=cos-2.0.30-slurm-healthy-compute
    ncn-m001# cray bos sessiontemplate describe $BOS_TEMPLATE --format json|jq '.boot_sets[] | select(.node_list)'
    ```
    
    If this query returns empty, then skip to sub-step 3.
    If this query returns with data, then one or more boot sets within the BOS Session template reference nodes explicitly by xname. Consider adding your new nodes to this list (sub-step 1) or adding them on the command line (sub-step 2).

    1. Adding new nodes to your list.
       
       1. Dump the current Session template.
          
          ```bash
          ncn-m001# cray bos sessiontemplate describe $BOS_TEMPLATE --format json > tmp.txt
          ```
       

>        2. Edit the tmp.txt file adding the new nodes to the node_list.

Reviewer: ```suggestion
       2. Edit the `tmp.txt` file, adding the new nodes to the node_list.
```

          
          ```bash
          ncn-m001# vi tmp.txt
          ```

    2. Create the Session template.
       

>        1. The name of the Session template is determined by the name provided to the '--name' option on the command line. Use the current value of $BOS_TEMPLATE if you want to overwrite the existing Session template. If you want to use the current value, skip this sub-step and go on to sub-step 2. Otherwise, provide a different name for BOS_TEMPLATE which will be used the '--name' option. The name specified in tmp.txt is overridden by the value provided by the '--name' option.

Reviewer: ```suggestion
       1. The name of the Session template is determined by the name provided to the `--name` option on the command line. Use the current value of `$BOS_TEMPLATE` if you want to overwrite the existing Session template. If you want to use the current value, skip this sub-step and go on to sub-step 2. Otherwise, provide a different name for `BOS_TEMPLATE` which will be used with the `--name` option. The name specified in `tmp.txt` is overridden by the value provided by the `--name` option.
```

>        ```bash

Reviewer: ```suggestion
           ```bash
```
